server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss

```diff
@@ -92,13 +92,27 @@ See the radio-option below as an example.
   }
 }
 
+// Styles common to tablet and mobile
 @media (max-width: 63.9em) {
   .usa-nav .usa-button,
   .usa-nav span {
     margin-top: 8px;
   }
 }
 
+// Styles for mobile
+@media (max-width: 48em) {
+  .usa-nav .usa-button,
```

Comment:
**question:** Do you need to repeat this margin style here since it's already in the `max-width: 63.9em` section?